wsgi.py

```diff
@@ -1,19 +1,17 @@
 import os
-import multiprocessing
 from gunicorn.app.base import BaseApplication
 from app import app as application
 
```

Comment:
@ltkum, @procrastinatio here I'm not sure if the disable is a good idea, we may want to implement the init() method from the BaseApplication as follow ?
```python
def init(self, parser, opts, args):
    pass
```
This would avoid the pylint warning and avoid an error if in the future the BaseApplication.load_config() method (which call the init()) is not overloaded.